src/components/Books.js

```diff
@@ -0,0 +1,25 @@
+import React, { useState } from 'react';
+
+const Book = () => {
+  const [book, setBook] = useState({
+    title: 'The Book Title',
+    author: 'The Book Author',
+  });
+
+  const deleteBook = () => {
+    setBook({
+      title: '',
+      author: '',
+    });
+  };
+
+  return (
+    <div>
+      <h2>{book.title}</h2>
+      <p>{book.author}</p>
+      <button type="button" onClick={deleteBook}>Delete Book</button>
+    </div>
+  );
+};
```

Comment:
- [ ] Please remove lines no. 9 to 14 because this is not a requirement for this milestone :+1: 
- [ ] Also, remove lines no. 4 to 7 and pass `book` as props :slightly_smiling_face: 
- [ ] Also, remove the `onClick` from button because this is not a requirement for this milestone :+1:
- [ ] Check below  :point_down:  my suggestions :slightly_smiling_face: 

```suggestion
const Book = ({book}) => {
  return (
    <div>
      <h2>{book.title}</h2>
      <p>{book.author}</p>
      <button type="button">Delete Book</button>
    </div>
  );
};
```